salt/hostname/init.sls

```diff
@@ -0,0 +1,11 @@
+{% set hostname = grains['id'] + '.' + pillar['internal_infra_domain'] %}
+
+/etc/hostname:
```

Comment:
Is this `file.managed` necessary considering that we are also running `hostnamectl --static`? I thought that would also update `/etc/hostname`...